packages/web/src/hooks/common/use-slippage.ts

```diff
@@ -0,0 +1,29 @@
+import { useAtom } from "jotai";
+import { CommonState } from "@states/index";
+import { useCallback } from "react";
+import { toDecimalNumber } from "@utils/number-utils";
+import { DEFAULT_SLIPPAGE } from "@constants/option.constant";
+
+export const useSlippage = () => {
+  const [slippage, setSlippage] = useAtom(CommonState.slippage);
+
+  const changeSlippage = useCallback(
+    (slippage: number) => {
+      let changedSlippage = slippage;
+      if (slippage < 0) {
+        changedSlippage = 0;
+      } else if (slippage > 100) {
+        changedSlippage = 100;
+      }
+      changedSlippage = toDecimalNumber(changedSlippage, 2);
```

Comment:
Does `slippage` need to `bignumber`?

Reply:
No, I wanted to round to the second decimal place.
But this seems to be wrong.